src/components/plain-button/PlainButton.js

```diff
@@ -22,7 +31,7 @@ const PlainButton = ({ children, className = '', getDOMRef, isDisabled = false,
 
     return (
         // eslint-disable-next-line react/button-has-type
-        <button className={`btn-plain ${className}`} ref={getDOMRef} type={type} {...rest} {...buttonProps}>
+        <button className={`btn-plain ${className}`} ref={getDOMRef} type={type} id={id} {...rest} {...buttonProps}>
```

Comment:
`id` would already be passed by the `...rest` here, no need to add it

Reply:
Fixed.